pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -228,58 +231,56 @@ stdenv.mkDerivation {
         *) echo "Multiple dynamic linkers found for platform '${targetPlatform.config}'." >&2;;
       esac
 
-      if [ -n "''${dynamicLinker:-}" ]; then
+      if [ -n "''${dynamicLinker-}" ]; then
```

Comment:
I'm mainly curious... this dash is some kind of special operator?  I can't find this one in `man bash`.

Reply:
@vcunat See https://wiki.bash-hackers.org/syntax/pe#use_a_default_value 

> If you omit the `:` (colon), like shown in the second form, the default value is only used when the parameter was unset, not when it was empty.

Reply:
Now I read that `man bash` part again &ndash; more properly &ndash; and I see it there as well :facepalm: